java/modules/basic/src/main/java/io/v6d/modules/basic/arrow/TableBuilder.java

```diff
@@ -87,4 +89,43 @@ public ObjectMeta seal(Client client) throws VineyardException {
 
         return client.createMetaData(meta);
     }
+
+    public static ObjectMeta fromRecordMeta(Client client, ObjectMeta schemaMeta, List<ObjectMeta> recordBatchMetas, int columns) throws VineyardException {
```

Comment:
fromRecordMeta -> fromRecordBatchMeta?

We don't have the concept of "record".

Reply:
OK